examples/misc_controls_arcball.html

```diff
@@ -0,0 +1,438 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - arcball controls</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+
+	</head>
+
+	<body>
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> - interactive controls comparison<br/>
+			<a href="http://www.polycount.com/forum/showthread.php?t=130641" target="_blank" rel="noopener">Cerberus(FFVII Gun) model</a> by Andrew Maximov.
+		</div>
+
+		<script type="module">
+			import * as THREE from '../build/three.module.js';
+
+			import { GUI } from './jsm/libs/dat.gui.module.js';
+
+			import { ArcballControls } from './jsm/controls/ArcballControls.js';
+			import { OrbitControls } from './jsm/controls/OrbitControls.js';
+			import { TrackballControls } from './jsm/controls/TrackballControls.js';
```

Comment:
In order to simplify the example, I suggest you remove both `OrbitControls` and `TrackballControls`.

At the homepage, the control examples just showcase a single control class for simplicity.